web/src/beta/ui/fields/InputSelectField.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  Selector,
+  SelectorProps,
+  TextInput,
+  TextInputProps
+} from "@reearth/beta/lib/reearth-ui";
+import { styled } from "@reearth/services/theme";
+import { FC, useState } from "react";
+
+import CommonField, { CommonFieldProps } from "./CommonField";
+
+export type InputFieldProps = CommonFieldProps &
+  Pick<
+    TextInputProps,
+    "value" | "placeholder" | "onChange" | "onBlur" | "disabled" | "appearance"
+  > &
+  Pick<SelectorProps, "displayValue"> & { sketchCustomProperties?: string[] };
+//
+
+const InputSelectField: FC<InputFieldProps> = ({
+  title,
+  description,
+  placeholder,
+  onBlur,
+  sketchCustomProperties,
+  displayValue
+}) => {
+  const sketchCustomPropertiesOption = sketchCustomProperties?.map(
+    (property) => ({
+      value: property,
+      label: property
+    })
+  );
+
+  const [selectValue, setSelectValue] = useState<string | undefined>(undefined);
+
+  const handleOnBlur = () => {
+    if (onBlur && typeof selectValue === "string") {
+      onBlur(selectValue);
+    }
+  };
```

Comment:
We need to support user input as well. When input blur we should update local value and call prop onBlur with new value